### PR TITLE
Add Beats known issue caused by credentials error with S3+SQS

### PR DESCRIPTION
**Pull request by dedemorton**
Adds Beats docs related to https://github.com/elastic/observability-docs/issues/2164.

Related Elastic Agent PR: https://github.com/elastic/observability-docs/pull/2168

@aspacca @kaiyan-sheng Please review this PR, too. I am pretty sure I am not fully understanding all the details here, but wanted to get a PR up so we can discuss. Thanks!

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -7,6 +7,52 @@
 === Beats version 8.4.1
 https://github.com/elastic/beats/compare/v8.4.0\...v8.4.1[View commits]
 
+==== Known Issue 
+
+// tag::credentials-error[]
+
+*Filebeat*
+
+Filebeat agents configured to read from S3 or SQS may return the following
```
Comment (kaiyan-sheng) [inline CHANGELOG.asciidoc:16]:
```suggestion
Filebeat agents configured to read from S3 input or S3 input with SQS notification setup may return the following
```

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -7,6 +7,52 @@
...
+----
+sqs ReceiveMessage failed: operation error SQS: ReceiveMessage, https response
+error StatusCode: 403, RequestID: cb57783a-505f-5099-9160-23b8eea8ddbb,
+api error SignatureDoesNotMatch: Credential should be scoped to a valid region. 
+----
+
+This error was introduced by a breaking change in the AWS library.
+
+Suggested resolution: In the Filebeat configuration, if an AWS input or module
+configuration sets `endpoint` to `amazonaws.com`, set it to an empty string
```
Comment (kaiyan-sheng) [inline CHANGELOG.asciidoc:29]:
```suggestion
configuration sets `endpoint` to `amazonaws.com`, set it to an empty string or remove this config parameter
```

File: CHANGELOG.asciidoc [hunk 3]
```diff
@@ -7,6 +7,52 @@
...
+instead. Also make sure the default AWS region is set in an environment
+variable, credentials or instance profile, or in the `default_region` setting in
+the configuration. For example:
+
+[source,yaml]
+----
+filebeat.inputs:
```
Comment (kaiyan-sheng) [inline CHANGELOG.asciidoc:36]: or can also be:
```
filebeat.inputs:
- type: aws-s3
...
  default_region: us-east-1
```

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (aspacca) [inline CHANGELOG.asciidoc:16]: leaving here similar comment as ones at https://github.com/elastic/observability-docs/pull/2168
I would be "less" specific: meaning is a problem related to any AWS metricset/fileset and not specific to S3 and SQS.

File: CHANGELOG.asciidoc [hunk 4]
```diff
@@ -7,6 +7,52 @@
...
+// tag::credentials-error[]
+
+*Filebeat*
+
+Filebeat agents configured to read from S3 or SQS may return the following
+error:
+
+[source,shell]
+----
+sqs ReceiveMessage failed: operation error SQS: ReceiveMessage, https response
```
Comment (aspacca) [inline CHANGELOG.asciidoc:21]: beware that the error message might differ for some users: it's anyway most of the time related to authentication and/or region

File: CHANGELOG.asciidoc (hunk 2 above)
Comment (aspacca) [inline CHANGELOG.asciidoc:29]:
```suggestion
configuration sets `endpoint` to a non empty string, set it to an empty string
```

File: CHANGELOG.asciidoc [hunk 5]
```diff
@@ -7,6 +7,52 @@
...
+error StatusCode: 403, RequestID: cb57783a-505f-5099-9160-23b8eea8ddbb,
+api error SignatureDoesNotMatch: Credential should be scoped to a valid region. 
+----
+
+This error was introduced by a breaking change in the AWS library.
+
+Suggested resolution: In the Filebeat configuration, if an AWS input or module
+configuration sets `endpoint` to `amazonaws.com`, set it to an empty string
+instead. Also make sure the default AWS region is set in an environment
+variable, credentials or instance profile, or in the `default_region` setting in
```
Comment (aspacca) [inline CHANGELOG.asciidoc:31]: 👍

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (dedemorton) [inline CHANGELOG.asciidoc:16]:
```suggestion
Filebeat agents configured to read from AWS inputs may return the following
```

File: CHANGELOG.asciidoc [hunk 6]
```diff
@@ -7,6 +7,52 @@
...
 https://github.com/elastic/beats/compare/v8.4.0\...v8.4.1[View commits]
 
+==== Known Issue 
+
+// tag::credentials-error[]
+
+*Filebeat*
+
+Filebeat agents configured to read from S3 or SQS may return the following
+error:
```
Comment (dedemorton) [inline CHANGELOG.asciidoc:17]:
```suggestion
Filebeat agents configured to read from AWS inputs may return an error similar to the following:
```

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (dedemorton) [reply CHANGELOG.asciidoc]: I applied Andrea's recommendation to make this more generic instead.

File: CHANGELOG.asciidoc [hunk 7]
```diff
@@ -7,6 +7,51 @@
...
+
+[source,yaml]
+----
+filebeat.inputs:
+- type: aws-s3
+...
+  endpoint: ""
+  default_region: us-east-1
+----
+
```
Comment (dedemorton) [inline CHANGELOG.asciidoc:41]:
```suggestion
...
  endpoint: "" <1>
  default_region: us-east-1
----
<1> You can set this value to an empty string or remove the configuration setting.
```

File: CHANGELOG.asciidoc [hunk 8]
```diff
@@ -7,6 +7,51 @@
...
+[source,yaml]
+----
+s3access:
+    enabled: false
+...
+    var.endpoint: ""
+    var.default_region: us-east-1
+----
+
```
Comment (dedemorton) [inline CHANGELOG.asciidoc:52]:
```suggestion
    var.endpoint: "" <1>
    var.default_region: us-east-1
----
<1> You can set this value to an empty string or remove the configuration setting

```

File: CHANGELOG.asciidoc [hunk 9]
```diff
@@ -7,6 +7,52 @@
...
+[source,yaml]
+----
+filebeat.inputs:
+- type: aws-s3
+...
+  endpoint: "" <1>
+  default_region: us-east-1
+----
+<1> You can set this value to an empty string or remove the configuration setting.
+Or for modules:
```
Comment (dedemorton) [inline CHANGELOG.asciidoc:42]:
```suggestion
<1> You can set this value to an empty string or remove the configuration setting.

Or for modules:
```

**Comment by dedemorton** [comment]
Is this good to merge?

**Comment by dedemorton** [comment]
Note addition was approved here: https://github.com/elastic/observability-docs/issues/2164#issuecomment-1242819501

**Pull request merged**